homework1/styles.css

```diff
@@ -0,0 +1,652 @@
+@font-face {
+  font-family: Baloo-Regular;
+  src: url(./fonts/Baloo-Regular.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Regular;
+  src: url(./fonts/Baloo2-Regular.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Medium;
+  src: url(./fonts/Baloo2-Medium.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Regular;
+  src: url(./fonts/Baloo2-Regular.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Medium;
+  src: url(./fonts/Baloo2-Medium.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-SemiBold;
+  src: url(./fonts/Baloo2-SemiBold.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Bold;
+  src: url(./fonts/Baloo2-Bold.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  -webkit-tap-highlight-color: transparent;
+}
+body {
+  font-family: "Baloo2-Regular", sans-serif;
+  line-height: 1.6;
+  margin: 0 auto;
+  min-height: 100vh;
+}
+
+.header--content,
+.main-nav {
+  display: flex;
+}
+
+.header--content {
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  padding: 2em 0;
+  background-color: #ffffff;
+  max-height: none;
+}
+
+.header--content .nav-icon {
+  display: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+h2,
+h3,
+a {
+  color: #000000;
+}
+
+a {
+  text-decoration: none;
+  font-weight: 500;
+}
+
+.main-nav {
+  margin-top: 5px;
+}
+
+.main-nav a {
+  padding: 10px 15px;
+  text-align: center;
+}
+
+.main-nav li:last-child a {
+  padding: 5px 30px;
+  margin-left: 20px;
+}
+
+.main-nav a {
+  color: #000000;
+  font-size: 1em;
+  font-family: Baloo2-Regular;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.main-nav a:hover {
+  color: #000000;
+  font-family: Baloo2-Bold;
+}
+
+.main-nav li:last-child a {
+  color: #ffffff;
+}
+
+/* menu icon */
+
+.header--content .menu-icon {
```

Comment:
for mobile menu i have a vertical scroll. Try to fix it

![image](https://user-images.githubusercontent.com/28695044/198210079-56823415-0384-4c9b-8ce0-a4c9cf891abe.png)